### PR TITLE
[openshift] Replace version  with a variable in docs link

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Replace fixed version number like `4.10` with a place holder variable
`OPENSHIFT_DOCS_VERSION` to make it possible to customize the docs link later.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml [hunk 1]
```diff
@@ -197,7 +197,7 @@ spec:
     - ClusterTasks based on Tekton Catalog 0.28
 
     ## Getting Started
-    In order to get familiar with _OpenShift Pipelines_ concepts and create your first pipeline, follow the [OpenShift Pipelines Docs](https://docs.openshift.com/container-platform/latest/pipelines/creating-applications-with-cicd-pipelines.html).
+    In order to get familiar with _OpenShift Pipelines_ concepts and create your first pipeline, follow the [OpenShift Pipelines Docs](https://docs.openshift.com/container-platform/OPENSHIFT_DOCS_VERSION/cicd/pipelines/creating-applications-with-cicd-pipelines.html).
```
Comment (savitaashture) [inline operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:200]: from where do we pass version info for `OPENSHIFT_DOCS_VERSION` variable

**Review by concaf — APPROVED** [review]
/lgtm

File: operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml (hunk 1 above)
Comment (nikhil-thomas) [reply operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:200]: during downstream productization or before before bundle generation

**Pull request merged**